backend/app/converters/lib/eac_base_map.rb

```diff
@@ -1114,7 +1114,7 @@ def name_map(type)
 
         case type
         when 'person'
-          nom_parts = val.split(',', 2)
+          nom_parts = val.split(', ', 2)
```

Comment:
```suggestion
          nom_parts = val.split(/,\s*/, 2)
```

Reply:
@lorawoodford I wonder if we want this to catch all cases like "a,b" "a, b" "a,   b"

Reply:
@quoideneuf Yup, makes sense to me.